### DISCUSSION TITLE
Welcome to ticketing-microservices-node-nextjs-react Discussions!

### DISCUSSION THREAD

**Discussion by VictorHugoAguilar** [VictorHugoAguilar/ticketing-microservices-node-nextjs-react#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VHAA1718** [comment]
Where we started collaborating

**Reply by VictorHugoAguilar** [reply]
let's start with the jwt

**Comment by VictorHugoAguilar** [comment]
Starting with obtaining the data of the authenticated user, creating the jwt

**Reply by VHAA1718** [reply]
Ok, what can I wear then?

**Reply by VictorHugoAguilar** [reply]
If you want, I'll write you privately now and we'll plan the issue

**Reply by VHAA1718** [reply]
zoom ¿?

**Reply by VictorHugoAguilar** [reply]
ok
